packages/components/src/SVG/SVG.tsx

```diff
@@ -1,79 +1,60 @@
 import React, { ReactNode, SVGAttributes } from "react"
 import classnames from "classnames"
 import { OverrideClassName } from "~types/OverrideClassName"
-import { warn } from "~utils/console"
 import styles from "./SVG.module.scss"
 
-type RolesType =
-  | "img" // meaningful, title should be read aloud to users who can't see it
-  | "presentation" // decorative, should be silent to users who can't see it
-
-export type SVGProps = {
-  children: ReactNode
+type BaseSVGProps = {
   inheritSize?: boolean
-  role?: RolesType
-  title?: string
-  desc?: string
 } & OverrideClassName<SVGAttributes<SVGElement>>
 
+type MeaningfulIcon = {
+  role: "img"
+  "aria-label": string
+} & OverrideClassName<SVGAttributes<SVGElement>>
```

Comment:
Only goes on the Base :)

```suggestion
}
```

Reply:
Oops, screwed it up on the rebase